src/fusionengine/files/math.py

```diff
@@ -1,4 +1,6 @@
 class Math:
     def __init__(self):
         self.PI = 3.141592653589793238462643383279502884197

```

Comment:
Python only renders *15 of 39* decimal digits with the print function, which means only 15 decimal digits actually survive in calculations. This means that `SMALLERPI` makes more sense to be `PI`. This is the pi in the `math` module, more commonly known as the standard math lib.